src/UI/Component/Panel/Sub.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel;
+
+/**
+ * This describes a Sub Panel.
+ */
+interface Sub extends Panel {
```

Comment:
Is the rule each component should represent an interface only one way or two way? From your comment I gather it is one way. What name would you chose for this interface?